README.md

```diff
@@ -308,6 +308,9 @@ See [CONTRIBUTING.md](https://github.com/pantheon-systems/pantheon-advanced-page
 
 ## Changelog ##
 
+## 1.3.0-dev (April 13, 2023) ##
```

Comment:
```suggestion
## Latest ##
```
Keep this to `Latest` in `develop` and feat branches and change in the release PR.